bedrock/firefox/views.py

```diff
@@ -337,11 +337,15 @@ def all_downloads(request, channel):
 
 @never_cache
 def firefox_os_geo_redirect(request):
-    country = get_country_from_request(request)
-    version = settings.FIREFOX_OS_COUNTRY_VERSIONS.get(
-        country,
-        settings.FIREFOX_OS_COUNTRY_VERSIONS['default']
-    )
+    locale = l10n_utils.get_locale(request)
+    if locale == 'en-US':
```

Comment:
it's actually `waffle.switch_is_active()`. There are examples elsewhere in this file if you need.